encord/client.py

```diff
@@ -612,20 +618,47 @@ def add_users(self, user_emails: List[str], user_role: ProjectUserRole) -> List[
 
         return list(map(lambda user: ProjectUser.from_dict(user), users))
 
-    def copy_project(self, copy_datasets=False, copy_collaborators=False, copy_models=False) -> str:
+    def copy_project(
+        self,
+        copy_datasets: Union[bool, CopyDatasetOptions] = False,
+        copy_collaborators=False,
+        copy_models=False,
+        *,
+        copy_labels: Optional[CopyLabelsOptions] = None,
+        new_title: str,
```

Comment:
Can this signature mirror the other `copy_project` signature? Also with the `new_title`. A default should really be added in this implementation, just in case someone is using the EncordClientProject. 

You can still just mention in the Args doc that there will be a default if unspecified.

Reply:
What would you consider a good default?

Reply:
The default you have was good. Just have here the same (i.e. `Optional[str] = None`) and have the `Copy of {title}` in here rather in the project, so the behaviour is the same between the two.

Reply:
So here we don't really have the `title`, only the `resource_id` I think

Reply:
Mhh let's chat about this. In general, users should be able to use the `EncordProjectClient` just as  they use the `Project` class